tests/environments/storage/test_local_storage.py

```diff
@@ -139,3 +139,17 @@ def test_add_flow_with_weird_name_is_cleaned():
         assert "!" not in loc
         assert " " not in loc
         assert "~" not in loc
+
+
+def test_build_healthchecks():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        s = Local(directory=tmpdir)
+        flow = Flow("TestFlow")
+        s.add_flow(flow)
+        assert s.build()
```

Comment:
This test doesn't seem to test anything related to healthchecks?

Reply:
a little indirect, but it is invoking the new healthcheck method (https://github.com/PrefectHQ/prefect/blob/storage_healthchecks/src/prefect/environments/storage/local.py#L118)

Reply:
Yeah it is indirect. This tests that the build passes w/ no healthcheck errors